app/views/admin/invitations/index.html.erb

```diff
@@ -5,7 +5,16 @@
         <h1 class="crayons-title ml-3 l:ml-0">Members<span class="screen-reader-only"> (Invitations)</span></h1>
         <%= render "admin/users/index/tabs" %>
       </div>
-      <%= render "admin/users/index/controls" %>
```

Comment:
Originally, the intention was to generalise this `controls` partial so we could use it in every tab. When I got stuck into this though I realised how many conditions we needed to account for if we did that (e.g. what we pass to the pagination, the labels for the search, the showing/hiding of the filter by role), when all we really want to share is the search input and pagination widgets which are _already_ their own partials.

So in the end, I think the better approach is re-using those lower level partials instead. It keeps everything much more readable and stops any unneeded entanglement